contracts/examples/bonding-curve/src/tokens/fungible_token.rs

```diff
@@ -0,0 +1,191 @@
+elrond_wasm::imports!();
+elrond_wasm::derive_imports!();
+
+use crate::{
+	events,
+	function_selector::FunctionSelector,
+	function_selector::SupplyType,
+	function_selector::{CurveArguments, Token},
+	storage,
+};
+
+const TOKEN_NUM_DECIMALS: usize = 0;
+#[elrond_wasm_derive::module]
+pub trait FTModule: storage::StorageModule + events::EventsModule {
+	// Upon issue the storage setting of the CurveArguments is done, reason why
+	// the supply details and the token accepted as payment are requested
+	#[payable("EGLD")]
+	#[endpoint(ftIssue)]
+	fn ft_issue(
```

Comment:
either include `fungible` or remove the prefix. the module encapsulates fungible token ops